client/src/components/HistoricElementView/HistoricElementView.js

```diff
@@ -27,9 +27,9 @@ const ElementalAreaHistoryFactory = (FieldGroup) =>
           {legend}
           <div className={'elemental-preview elemental-preview--historic'}>
             {data.ElementEditLink &&
-              <a className={'elemental-preview__link'} href={data.ElementEditLink}>
-                {i18n._t('HistoricElementView.BLOCK_HISTORY', 'View block')}
-                <i className={'font-icon-angle-right'} />
+              <a className="elemental-preview-link" href={data.ElementEditLink}>
```

Comment:
The CSS classname change could be interpreted as a breaking change. I don't see a problem with the way it was - perhaps you could revert that part?

Reply:
Fixed.